spec/manticore/client_spec.rb

```diff
@@ -245,6 +245,23 @@
         end
       end
 
+      context 'when client_key ending on whitespace character' do
+        let(:client) do
+          Manticore::Client.new(
+            ssl: {
+              verify: :default,
+              ca_file: File.expand_path('../ssl/root-ca.crt', __dir__),
+              client_cert: File.read(File.expand_path('../ssl/client_whitespace.crt', __dir__)),
+              client_key: File.read(File.expand_path('../ssl/client_whitespace.key', __dir__))
+            }
+          )
+        end
+
+        it 'doesn\'t throw an Exception' do
```

Comment:
Perhaps we can say it without apostrophes?

```suggestion
        it 'no exception is thrown' do
```